applications/nrf_desktop/src/modules/hid_state.c

```diff
@@ -386,7 +392,22 @@ static void clear_items(struct items *items)
 {
 	memset(items->item, 0, sizeof(items->item));
 	items->item_count = 0;
-	items->update_needed = false;
+}
+
+static void clear_axes(struct axis_data *axes)
```

Comment:
You clear axis_count here too, is it intended?

Reply:
no, but let's fix it in the next PR. it is not used yet